cryptoservice/crypto_service_test.go

```diff
@@ -115,20 +118,18 @@ func (c CryptoServiceTester) TestSignWithKey(t *testing.T) {
 // asserts that signing, if there are no matching keys, produces no signatures
 func (c CryptoServiceTester) TestSignNoMatchingKeys(t *testing.T) {
 	cryptoService := c.cryptoServiceFactory()
-	content := []byte("this is a secret")
 
 	privKey, err := trustmanager.GenerateECDSAKey(rand.Reader)
 	assert.NoError(t, err, c.errorMsg("error creating key"))
 
-	// Test Sign with key that is not in the cryptoservice
-	signatures, err := cryptoService.Sign([]string{privKey.ID()}, content)
-	assert.NoError(t, err, c.errorMsg("signing failed"))
-	assert.Len(t, signatures, 0, c.errorMsg("wrong number of signatures"))
+	// Test Sign
+	_, _, err = cryptoService.GetPrivateKey(privKey.ID())
+	assert.Error(t, err, c.errorMsg("Should not have found private key"))
 }
 
 // If there are multiple keystores, even if all of them have the same key,
 // only one signature is returned.
```

Comment:
Oops sorry, forgot about this comment.  Could we please change this to say "GetPrivateKey succeeds even if there are multiple keystores with the same key" or somesuch?